endpoints/openrtb2/amp_auction.go

```diff
@@ -443,7 +441,11 @@ func (deps *endpointDeps) overrideWithParams(ampParams amp.Params, req *openrtb2
 		req.Imp[0].TagID = ampParams.Slot
 	}
 
-	policyWriter, policyWriterErr := readPolicy(ampParams.Consent)
+	if err := setRegExtGDPR(ampParams.GdprApplies, req); err != nil {
+		return []error{err}
+	}
```

Comment:
Can we incorporate the gdpr field into the ReadPolicy method?

Reply:
Done